.github/workflows/python-package.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python package
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Why only these versions?

Reply:
As mentioned, that is what the template suggested. I had a look into the project.toml and the setup.py. I did not see any python version mentioned. So I left the templates default.

Reply:
Let's aim to test all versions that are not end of life at the moment, i.e. add 3.12 and 3.13 to the list :)